core/domain/summary_services.py

```diff
@@ -63,6 +63,16 @@ def get_displayable_exp_summary_dicts_matching_ids(exploration_ids):
     displayable_exp_summaries = []
     exploration_summaries = (
         exp_services.get_exploration_summaries_matching_ids(exploration_ids))
+    displayable_exp_summaries = get_displayable_exp_summary_dicts(
```

Comment:
Don't overwrite displayable_exp_summaries -- just do:

    return get_displayable_exp_summary_dicts(exploration_summaries)

and drop the unnecessary variable assignment on line 63.